### PR TITLE
Documentation improvements prompted by #160 & #168

### PR DESCRIPTION
**Pull request by wcjordan**
## Description
Documentation improvements prompted by #160 & #168 
Update documentation to meet current state and point users to useful resources

## Motivation and Context
Our documentation is stale and misleading in places.  Make things clear for new developers regarding our state and how to get started.

## How Has This Been Tested?
N/A

## Screenshots (if appropriate):

https://github.com/wcjordan/fixed-data-table-2
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +34,13 @@ Install `fixed-data-table-2` using npm.
 ```shell
 npm install fixed-data-table-2
 ```
-Add the default stylesheet `dist/fixed-data-table.css`, then import it into any module.
+Add the default stylesheet `dist/fixed-data-table.css` using a link tag or import it with a CSS module.
 
 ### Basic Example
-
+Note this example demonstrates typical FDT usage, but won't work out of the box.
```
Comment (KamranAsif) [inline README.md]: This seems excessive. Our library should be targeted towards normal react devs. If someone doesn't know how to compile JSX and how to set up a react app probably isn't ready to use this.

Comment (wcjordan) [inline README.md]: Sure, do you think we should pare it down but leave some pointers to the pre-requisite knowledge?  Anything stand out as important vs can be removed?

Comment (KamranAsif) [inline README.md]: I think we should only include the MyCustomCell that you have. Maybe show the code inline. The rest of this seems excessive

Comment (wcjordan) [inline README.md]: How about renaming the section to Code Sample and re-wording to

> This is a doesn't include a definition for MyCustomCell and won't work out of the box.  For more detailed examples, please see the examples section of the documentation.  If you need help getting started with a React build system, we recommend create-react-app.

Comment (KamranAsif) [inline README.md]: sgtm

**Pull request merged**